environment

```diff
@@ -107,6 +107,9 @@ GOOGLE_APPLICATION_CREDENTIALS_SECRETS_NAME="gcp-credentials.json"
 GOOGLE_APPLICATION_SECRETS_SECRETS_NAME="application_secrets.json"
 ES_ALLOWED_SOURCE_IP_SECRETS_NAME="es_source_ip"
 DSS_DEBUG=0
+TF_VAR_PROJECT=dcp
```

Comment:
Update `https://github.com/HumanCellAtlas/data-store/blob/master/docs/environment/README.md` for each.

Reply:
shall do

Reply:
If we move away from using a TF module, we can give these variables canonical DSS names, e.g. `DSS_INFRA_TAG_PROJECT`.